Runtime panic when trying to Ctrl+C out of pulling ai/qwen3:latest

**Describe the bug**

When pulling `ai/qwen3:latest`, using <kbd>Ctrl+C</kbd> appears to cause a panic.

**Version affected**

```
> cagent-windows-amd64.exe version
cagent version v1.15.8
Commit: dab39e4bf77d8fb9cb72429850dc186224204c8c
```

**How To Reproduce**

Detailed steps to reproduce the behavior:

1. Download `cagent-windows-amd64.exe`.
2. Run it.
3. Get prompted to pull `ai/qwen3:latest`.
4. Accept this.
5. Let it download a little and then <kbd>Ctrl+C</kbd>.
6. Go stack trace is printed.

**OS and Terminal type**

Windows 11 Pro 25H2 (26200.7171)

```
 cmd --version
Microsoft Windows [Version 10.0.26200.7171]
(c) Microsoft Corporation. All rights reserved.
```

Here's the stack trace.
```
C:\Users\RemySuen\Downloads>cagent-windows-amd64.exe run

Model ai/qwen3:latest not found locally.
Do you want to pull it now? ([y]es/[n]o): y
Pulling model ai/qwen3:latest...
b505f0cf6920: Downloading [==>                                                ]    272MB/5.028GB
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x7ff620e45957]

goroutine 1 [running]:
github.com/docker/cagent/pkg/model/provider/dmr.(*Client).ID(0x7ff6225752f0?)
        <autogenerated>:1 +0x17
github.com/docker/cagent/pkg/teamloader.getModelsForAgent-range1({0xc000298e80, 0x4})
        github.com/docker/cagent/pkg/teamloader/teamloader.go:216 +0x4b5
github.com/docker/cagent/pkg/teamloader.getModelsForAgent.SplitSeq.splitSeq.func1(...)
        strings/iter.go:59
github.com/docker/cagent/pkg/teamloader.getModelsForAgent({0x7ff6225752f0?, 0xc0005b6900?}, 0xc0005b6900?, 0x0?, 0x38?, 0xc0000097d0?)
        github.com/docker/cagent/pkg/teamloader/teamloader.go:182 +0x1a4
github.com/docker/cagent/pkg/teamloader.Load({0x7ff6225752f0, 0xc0005b6900}, {0x7ff62256a850, 0xc0002e0720}, 0xc0001ace58, {0xc00063f950, 0x1, 0x7ff620201425?})
        github.com/docker/cagent/pkg/teamloader/teamloader.go:120 +0xae9
github.com/docker/cagent/cmd/root.(*runExecFlags).loadAgentFrom(0x0?, {0x7ff6225752f0, 0xc0005b6900}, {0x7ff62256a850?, 0xc0002e0720?})
        github.com/docker/cagent/cmd/root/run.go:163 +0x92
github.com/docker/cagent/cmd/root.(*runExecFlags).runOrExec(0xc0001ace00, {0x7ff6225752f0, 0xc0005b6900}, 0xc0007e9b60, {0x7ff623a9d420, 0x0, 0x0}, 0x1)
        github.com/docker/cagent/cmd/root/run.go:139 +0x4ac
github.com/docker/cagent/cmd/root.(*runExecFlags).runRunCommand(0xc0001ace00, 0xc00014ef08, {0x7ff623a9d420, 0x0, 0x0})
        github.com/docker/cagent/cmd/root/run.go:88 +0xe5
github.com/spf13/cobra.(*Command).execute(0xc00014ef08, {0x7ff623a9d420, 0x0, 0x0})
        github.com/spf13/cobra@v1.10.2/command.go:1015 +0xb02
github.com/spf13/cobra.(*Command).ExecuteC(0xc000049808)
        github.com/spf13/cobra@v1.10.2/command.go:1148 +0x465
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.10.2/command.go:1071
github.com/spf13/cobra.(*Command).ExecuteContext(...)
        github.com/spf13/cobra@v1.10.2/command.go:1064
github.com/docker/cagent/cmd/root.Execute({0x7ff6225752f0, 0xc0005b6900}, {0x7ff622552e00, 0xc0000a4030}, {0x7ff622552e20, 0xc0000a4038}, {0x7ff622552e20, 0xc0000a4040}, {0xc0000a84b0, 0x1, ...})
        github.com/docker/cagent/cmd/root/root.go:132 +0x339
main.main()
        github.com/docker/cagent/main.go:15 +0xd9
```

--- a/pkg/teamloader/teamloader.go
+++ b/pkg/teamloader/teamloader.go
@@ -215,7 +215,6 @@ func getModelsForAgent(ctx context.Context, cfg *latest.Config, a *latest.AgentC
 		if err != nil {
 			return nil, err
 		}
-		model.ID()
 		models = append(models, model)
 	}
 

